homeassistant/components/onewire/sensor.py

```diff
@@ -94,13 +104,18 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up the one wire Sensors."""
     base_dir = config[CONF_MOUNT_DIR]
     owport = config[CONF_PORT]
-    owhost = config.get(CONF_HOST)
+    owhost = config[CONF_HOST]
```

Comment:
Host is optional without default in the platform schema.

Reply:
I don't understand what you mean.

Reply:
We need to use `dict.get` here, so please revert your change.